Makefile

```diff
@@ -0,0 +1,60 @@
+# Define variables
+ENV_FILE=.env
+ENV=local
+
+# Default target
+all: setup
+
+# Setup the application
+setup: install_deps set_env generate_env collect_static migrate setup_waffle_switches
+
+# Install dependencies
+install_deps:
+	poetry install
+	npm install
+
+# Set environment variable
+set_env:
+	@echo "Setting ENV to ${ENV}"
+	export ENV=${ENV}
+
+# Generate .env file
+generate_env:
+	op inject --in-file .env.tpl --out-file ${ENV_FILE}
+	@echo "Optionally, set CATALOGUE_TOKEN in ${ENV_FILE}"
+
+# Collect static files
+collect_static:
+	poetry run python manage.py collectstatic --noinput
+
+# Run migrations
+migrate:
+	poetry run python manage.py migrate
+
+# Setup waffle switches
+setup_waffle_switches:
+	poetry run python manage.py waffle_switch search-sort-radio-buttons off --create
+	poetry run python manage.py waffle_switch display-result-tags off --create
+
+# Run the application
+run:
+	poetry run python manage.py runserver
+
+# Run unit tests
+test: test_python test_js
+
+# Run Python unit tests
+test_python:
```

Comment:
You might want to consider adding the chrome-path options for the test
`poetry run pytest -vv --chromedriver-path /opt/homebrew/bin/chromedriver ./tests`

Reply:
Another consideration is whether this should work on Linux as well as MacOS

We probably do at some point, as then we can use it in our gihub actions workflows, but we could come back to that later